src/main/java/at/ac/tuwien/kr/alpha/common/BinaryOperator.java

```diff
@@ -1,8 +1,5 @@
 package at.ac.tuwien.kr.alpha.common;
 
-import at.ac.tuwien.kr.alpha.common.predicates.BuiltinBiPredicate;
-import at.ac.tuwien.kr.alpha.common.predicates.FixedInterpretationPredicate;
-
 public enum BinaryOperator {
```

Comment:
Now that we also have `ArithmeticOperator` I think that `BinaryOperator` is confusing and it should be renamed to `ComparisonOperator`.